src/controllers/controller.bar.js

```diff
@@ -112,10 +112,9 @@ module.exports = function(Chart) {
 			var me = this;
 			var meta = me.getMeta();
 			var yScale = me.getScaleForId(meta.yAxisID);
-			var base = yScale.getBaseValue();
-			var original = base;
+			var base = 0;
```

Comment:
I'm refactoring the bar controller and after a few tests, `var base = 0;` may be the correct way to calculate the bar positions! You can try it by setting a min on the scale. But that also means `return scale.getBasePixel()` is wrong.